docs/releasehistory.rst

```diff
@@ -7,6 +7,30 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 * ``minor`` increments add features but do not break API compatibility
 * ``micro`` increments represent bugfix releases or improvements in documentation
 
+0.6.1 - Bugfixes
+----------------
+
+New features
+""""""""""""
+- `PR #469 <https://github.com/openforcefield/openforcefield/pull/XXX>`_:
```

Comment:
While it's possible that this branch will come in on a different PR, I think we should assume this will come in, and just put the PR number right in the URL.

Reply:
Fixed in dbc1451